### PR TITLE
use new psr-15 interface packages

### PR DESCRIPTION
**Pull request by dirx**
prepared next 1.0.0 release.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ Stack implements ```Interop\Http\ServerMiddleware\DelegateInterface```.
...
 use Psr\Http\Message\ServerRequestInterface;
-use Interop\Http\ServerMiddleware\DelegateInterface;
-use Interop\Http\ServerMiddleware\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Psr\Http\Server\MiddlewareInterface;
 
 class TrickyMiddleware implements MiddlewareInterface
 {
-    public function process(ServerRequestInterface $request, DelegateInterface $frame) : ResponseInterface
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $frame) : ResponseInterface
```
Comment (harikt) [inline Readme.md]: why not name the `$frame` to `$handler` ? BC break ?

Comment (dirx) [reply Readme.md]: thank you for the hint - missed that one.

File: tests/StackTest.php [hunk 2]
```diff
@@ -50,30 +50,32 @@ public function testServerEmptyMiddleware()
...
-    public function testServerMiddlewareProcessingOrder()
+    public function testServerMiddlewareHandlingOrder()
     {
         $callCounter = 0;
 
         $middleware1 = $this->getMiddlewareMock();
         $middleware1->expects($this->once())
             ->method('process')
-            ->willReturnCallback(function (ServerRequestInterface $request, DelegateInterface $frame) use (&$callCounter
+            ->willReturnCallback(function (ServerRequestInterface $request, RequestHandlerInterface $frame) use (
```
Comment (harikt) [inline tests/StackTest.php]: The $frame is repeated on multiple other places. See one here. Another one below.

**Comment by harikt** [comment]
Good to me :+1:  .

**Pull request merged**